Add tenancy migration guidelines (#9509)
fishtown-analytics/docs.getdbt.com: website/docs/guides/tenancy-migration.md

hoverSnippet: "Self-serve guide for migrating between dbt platform tenancy types."
icon: 'guides'
hide_table_of_contents: true
tags: ['Migration', 'Tenancy', 'dbt platform']
keywords: ['tenancy migration', 'multi-tenant', 'single tenant', 'MT to ST', 'migrate tenant', 'cell-based']
level: 'Advanced'
unlisted: true
---

:::warning Contract change required
Migrating between tenancy types requires a contract change.

Make sure you contact your account manager before starting this process to confirm eligibility, timeline, and initiate the necessary contract updates. _Do not_ begin migration steps until a contract change is confirmed.
:::

## Overview

This guide walks you through a self-serve migration between [<Constant name="dbt_platform"/> tenancy types](/docs/platform/about-platform/tenancy) &mdash; for example, moving from multi-tenant to single-tenant.

It's meant for accounts that have been assessed by the dbt Labs team as low complexity. Your dbt Labs contact will provide a migration window before you begin &mdash; do not start until you've received it.

> Use the table below to confirm this guide is the right path for your account. Any one signal crossing the high threshold is enough to recommend a Professional Services-led migration instead.

Reviewer: ```suggestion
Use the following table to confirm this guide is the right path for your account. If your account crosses the high-complexity threshold for even one signal, we recommend a Professional Services-led migration instead.
```

<SimpleTable>

| Signal | Low complexity (self-serve) | High complexity (PS-led) |
|--------|----------------------------|--------------------------|
| Active jobs (run in last 30 days) | Fewer than 10 | 10 or more |
| Projects | Fewer than 3 | 3 or more |
| Environments | Fewer than 5 | 5 or more |

> | **What to do** | Follow this guide | Contact your account manager to scope a [Professional Services migration](https://www.getdbt.com/services) |

Author: ```suggestion
| **What to do** | Follow this guide or contact  [Professional Services migration](https://www.getdbt.com/services) for help | Contact your account manager to scope a [Professional Services migration](https://www.getdbt.com/services) |
```

> | **What to do** | Follow this guide or contact  [Professional Services migration](https://www.getdbt.com/services) for help | Contact your account manager to scope a [Professional Services migration](https://www.getdbt.com/services) |

Reviewer: ```suggestion
| **How to proceed** | If all signals are low complexity, follow this guide or contact  [Professional Services migration](https://www.getdbt.com/services) for help | If one or more signals are high complexity, contact your account manager to scope a [Professional Services migration](https://www.getdbt.com/services) |
```
